src/main/java/ValidationMessages_fr.properties

```diff
@@ -0,0 +1,49 @@
+user.firstName=Veuillez entrer votre pr�nom.
+user.lastName=Veuillez entrer votre nom de famille. 
+user.invalidEmail=Veuillez entrer une adresse courriel valide.
+user.enterUsername=Veuillez entrer un nom d'utilisateur.
+user.usernameLength=Le nom d'utilisateur doit comporter entre 2 et 60 caract�res.
+user.illegalCharacters=Caract�re(s) non valide(s) utilis�(s). Les caract�res valides sont a-Z, 0-9, '_', '-' et '.'.
+
+user.enterNickname=Veuillez entrer un pseudonyme.
+user.nicknameLength=Le pseudonyme ne peut exc�der 30 caract�res.
+user.nicknameNotnumber=Le pseudonyme ne devrait pas �tre un nombre
+
+dataset.templatename=Veuillez ajouter un nom pour le mod�le d'ensemble de donn�es.
+dataset.nameLength=Le nom ne peut exc�der 255 caract�res.
+
+dataverse.name=Veuillez entrer un nom.
+dataverse.alias=Veuillez entrer un alias.
+dataverse.aliasLength=L'alias ne peut exc�der 60 caract�res.
+dataverse.aliasNotnumber=L'alias ne devrait pas �tre un nombre.
+dataverse.nameIllegalCharacters=Caract�re(s) non valide(s) utilis�(s). Les caract�res valides sont a-Z, 0-9, '_', '-' et '.'.
+dataverse.category=Veuillez s�lectionner une cat�gorie pour votre dataverse.
+contenttype.slash=Le type de contenu doit contenir une barre oblique.
+setspec.notNumber=Le nom (Setspec) ne devrait pas �tre un nombre.
+setspec.maxLength=Le nom (Setspec) ne peut exc�der 30 caract�res.
+
+role.name=Un r�le doit avoir un nom.
+desc.maxLength=Le description ne peut exc�der 255 caract�res.
+alias.maxLength=L'alias ne peut exc�der 26 caract�res.
+alias.illegalCharacters=L'alias ne peut �tre vide. Les caract�res valides sont a-Z, 0-9, '_', '-'.
+
+custom.response=Veuillez entrer la r�ponse.
+custom.questiontext=Veuillez entrer le texte de la question.
+filename.illegalCharacters=Le nom du fichier ne peut contenir aucun des caract�res suivants\u00A0: \ / : * ? " < > | ; # .
```

Comment:
Is this `\u00A0` "NO-BREAK SPACE" unicode character necessary?

Reply:
I would say yes. Implementation for the typographic French rules specifies:

before  ---  punctuation sign  --- after
---------------------------------------------------------
U+00A0 NO-BREAK SPACE   --- : --- U+0020 SPACE 

References:
[OLF] (http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?t1=1&id=2039)  
[TERMIUM] (http://www.btb.termiumplus.gc.ca/redac-chap?lang=eng&lettr=chapsect6&info0=6.13#zz6.13)